Library/Homebrew/utils/github.rb

```diff
@@ -296,7 +296,9 @@ def search_code(**qualifiers)
   def issues_for_formula(name, options = {})
     tap = options[:tap] || CoreTap.instance
     tap_full_name = options[:tap_full_name] || tap.full_name
-    search_issues(name, state: "open", repo: tap_full_name, in: "title")
+    open_issues = search_issues(name, state: "open", repo: tap_full_name, in: "title")
+    open_issues unless options.key?(:version)
+    Array(open_issues) | Array(search_issues("#{name} #{options[:version]}", repo: tap_full_name, in: "title"))
```

Comment:
I'm not seeing a clean way to pull the logic out since it currently by default looks for open issues and `exceptions.rb` uses this. Should the default behavior be to search all issues unless a state is passed in as an option and `fetch_issues` can be changed to search only open issues?

Reply:
> Should the default behavior be to search all issues unless a state is passed in as an option and `fetch_issues` can be changed to search only open issues?

Sounds good 👍